File: go/v2/echo_service.go
package v2

import "errors"
import "test-service/spec/v2"

type EchoService struct{}

func (service *EchoService) EchoBody(body *v2.Message) (*v2.EchoBodyResponse, error) {
	return nil, errors.New("implementation has not added yet")
}

File: java-spring/src/main/java/test_service/services_implementations/v2/EchoService.java
package test_service.services_implementations.v2;

import java.math.BigDecimal;
import java.time.*;
import java.util.*;

import org.springframework.stereotype.Service;

import test_service.v2.models.*;
import test_service.v2.services.echo.*;

@Service("EchoServiceV2")
public class EchoService implements IEchoService {
	@Override
	public Message echoBody(Message body) {
		throw new UnsupportedOperationException("Implementation has not added yet");
	}
}
